### PR TITLE
Serialization: Implement CUS0, CUS1 and CUS2

### PR DESCRIPTION
**Pull request by christianguenter2**
#1114

### CODE REVIEW

File: src/zabapgit_object_cus1.prog.abap [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    DATA: ls_message TYPE hier_mess.
+
+    CALL FUNCTION 'S_CUS_ACTIVITY_DELETE'
+      EXPORTING
+        activity = mv_customizing_activity
+      IMPORTING
+        message  = ls_message.
+
+    IF ls_message-msgty <> 'S'.
+      zcx_abapgit_exception=>raise( |error from delete CUS0 { mv_customizing_activity } S_CUS_ACTIVITY_DELETE| ).
```
Comment (larshp) [inline src/zabapgit_object_cus1.prog.abap]: "CUS0" -> "CUS1"

File: src/zabapgit_object_cus1.prog.abap [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        customer_exit_implementation = ls_customzing_activity-activity_customer_exit-impl_name
+      IMPORTING
+        message                      = ls_message
+      TABLES
+        activity_title               = ls_customzing_activity-activity_title
+        objects                      = ls_customzing_activity-objects
+        objects_texts                = ls_customzing_activity-objects_title.
+
+    IF ls_message-msgty <> 'S'.
+      zcx_abapgit_exception=>raise( |error from deserialize CUS0 { mv_customizing_activity } S_CUS_ACTIVITY_SAVE| ).
```
Comment (larshp) [inline src/zabapgit_object_cus1.prog.abap]: "CUS0" -> "CUS1"

File: src/zabapgit_object_cus2.prog.abap [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    DATA: ls_message TYPE hier_mess.
+
+    CALL FUNCTION 'S_CUS_ATTRIBUTES_DELETE'
+      EXPORTING
+        img_attribute = mv_img_attribute
+      IMPORTING
+        message       = ls_message.
+
+    IF ls_message-msgty <> 'S'.
+      zcx_abapgit_exception=>raise( |error from delete CUS22 { mv_img_attribute } S_CUS_ATTRIBUTES_DELETE| ).
```
Comment (larshp) [inline src/zabapgit_object_cus2.prog.abap]: "CUS22" -> "CUS2"

File: src/zabapgit_object_cus2.prog.abap [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+        img_attribute         = ls_customizing_attribute-header
+      IMPORTING
+        message               = ls_message
+      TABLES
+        attributes_title      = ls_customizing_attribute-titles
+        attributes_countries  = ls_customizing_attribute-countries
+        attributes_components = ls_customizing_attribute-components.
+
+    IF ls_message-msgty <> 'S'.
+      zcx_abapgit_exception=>raise( |error from deserialize CUS22 { mv_img_attribute } S_CUS_ATTRIBUTES_SAVE| ).
```
Comment (larshp) [inline src/zabapgit_object_cus2.prog.abap]: "CUS22" -> "CUS2"

File: src/zabapgit_object_cus1.prog.abap (hunk 1 above)
Comment (christianguenter2) [reply src/zabapgit_object_cus1.prog.abap]: fixed

File: src/zabapgit_object_cus1.prog.abap (hunk 2 above)
Comment (christianguenter2) [reply src/zabapgit_object_cus1.prog.abap]: fixed

File: src/zabapgit_object_cus2.prog.abap (hunk 3 above)
Comment (christianguenter2) [reply src/zabapgit_object_cus2.prog.abap]: fixed

File: src/zabapgit_object_cus2.prog.abap (hunk 4 above)
Comment (christianguenter2) [reply src/zabapgit_object_cus2.prog.abap]: fixed

**Comment by christianguenter2** [comment]
fixed all

**Pull request merged**